app/menu/main-menu.js

```diff
@@ -0,0 +1,76 @@
+const {Menu} = require('electron');
+const electron = require('electron');
+const app = electron.app;
+
+const template = [
+  {
+    label: 'Edit',
+    submenu: [
+      {
+        role: 'undo'
+      },
+      {
+        role: 'redo'
+      },
+      {
+        type: 'separator'
+      },
+      {
+        role: 'cut'
+      },
+      {
+        role: 'copy'
+      },
+      {
+        role: 'paste'
```

Comment:
I did remove the Paste and Match Style option, so it sounds like you're getting the default Electron menu (the one that appears in developer mode). It looks similar because I based this on the example from the Electron docs which shows you how to make the same menu as the default developer menu. I'll take a look at the office today.